template.tpl

```diff
@@ -49,7 +49,7 @@ ___TEMPLATE_PARAMETERS___
         "type": "REGEX"
       }
     ],
-    "displayName": "Ad Account ID",
```

Comment:
I think the answer is "it's out of scope for now", but we're not changing the prefix on the pixel ID, right?  That is, we're still good-to-go on line 47 with the `(t2_|a2_)[a-z0-9]+` regex, right?

Reply:
I think the plan is to update the prefix for pixels.  We will always support both `t2` and `a2` as those ids will likely never change on old websites.  New pixels created once we have the full creation process will have a different prefix.

Reply:
correct! no changes to any IDs at the moment. And then as @creativefunction13 noted, when we do, it'll just be for new IDs, so the regex would be updated to also support whatever new scheme that ends up being while continuing to support the old ones